frontend/packages/dev-console/src/components/import/section/IconSection.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { useFormikContext, useField } from 'formik';
+import { FormGroup } from '@patternfly/react-core';
+
+import FormSection from './FormSection';
+import IconDropdown from '../icon/IconDropdown';
+
+const IconSection: React.FC = () => {
+  const [field] = useField<string>('runtimeIcon');
+  const formik = useFormikContext<{ runtimeIcon: string }>();
+
+  const onChanged = (value: string) => {
+    formik.setFieldValue('runtimeIcon', value);
+    formik.setFieldTouched('runtimeIcon');
+  };
+
+  return (
+    <div className="icon-section">
```

Comment:
Do you need this div? The class name isn't defined and has no prefix.

Reply:
Good catch. This is left over from moving styles from IconSection to the IconDropdown. Removed it now.